backend/api/Makefile

```diff
@@ -1,15 +1,36 @@
+# Makefile to generate KFP api clients from proto.
+
 IMAGE_TAG=kfp-api-generator
+# Contact one of KFP maintainers if this remote image needs an update.
```

Comment:
Should we refer to where the owner can be found?

Reply:
Good point!